### PR TITLE
Feature/stake with us

### PR DESCRIPTION
**Pull request by foodaka**
Sets up blog capabilities for new pages for generating content

### CODE REVIEW

File: src/components/common/Layout/Layout.js [hunk 1]
```diff
@@ -7,15 +7,17 @@ import SEO from '@common/SEO';
...
-const Layout = ({ children }) => (
-  <ThemeProvider theme={theme}>
-    <>
-      <SEO />
-      <GlobalStyles />
-      {children}
-    </>
-  </ThemeProvider>
-);
+const Layout = (props) => {
```
Comment (marbar3778) [inline src/components/common/Layout/Layout.js]: these changes can be undone

File: src/components/sections/Staked.js [hunk 2]
```diff
@@ -52,8 +55,9 @@ const Staked = () => (
           <div>
             <h1>Currently Staking on</h1>
             <LogoGrid>
-              {LOGOS.map(({ logo, link }) => (
-                <ExternalLink
+              {LOGOS.map(({ logo, link, page }) => (
+                <Link
+                  to={`/blog/${page}`}
```
Comment (foodaka) [inline src/components/sections/Staked.js]: TODO: fix for kusama to just redirect to kusama network for now

File: src/pages/stake-cosmos.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (foodaka) [inline src/pages/stake-cosmos.js]: TODO DELETE FILE

File: src/components/common/Layout/Layout.js (hunk 1 above)
Comment (foodaka) [reply src/components/common/Layout/Layout.js]: 029113595b79c9f79e882075cc16898de1cc68d0

File: src/components/sections/Staked.js (hunk 2 above)
Comment (foodaka) [reply src/components/sections/Staked.js]: 029113595b79c9f79e882075cc16898de1cc68d0

File: src/pages/stake-cosmos.js (hunk 3 above)
Comment (foodaka) [reply src/pages/stake-cosmos.js]: 137eda61578927ca1a7926e4cc509de984b90915

File: gatsby-node.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      // createFilePath returns a path with the leading "/".
+      value: `/blog${value}`,
+    })
+  }
+}
+
+exports.createPages = async ({ graphql, actions, reporter }) => {
+  // Destructure the createPage function from the actions object
+  const { createPage } = actions
+  const result = await graphql(`
```
Comment (marbar3778) [inline gatsby-node.js]: are we using this?

File: src/components/sections/About.js [hunk 5]
```diff
@@ -74,13 +74,14 @@ const About = () => (
             <div>
               <h2>Build the Future</h2>
               <p>
-                We believe in helping to build out a future we would want ot live in. 
-                To help accomplish this goal we will be allotting 10% of our profits towards
-                OSS (open source software) this is not only limited to blockchain projects, beach and 
-                forest cleanups, and educating more people.
+                We believe in helping to build out a future we would want ot
```
Comment (marbar3778) [inline src/components/sections/About.js]:
```suggestion
                We believe in helping to build out a future we would want to
```

File: src/components/sections/About.js [hunk 6]
```diff
@@ -74,13 +74,14 @@ const About = () => (
...
+                live in. To help accomplish this goal we will be allotting 10%
+                of our profits towards OSS (open source software) this is not
+                only limited to blockchain projects, beach and forest cleanups,
```
Comment (marbar3778) [inline src/components/sections/About.js]:
```suggestion

```

File: src/components/sections/About.js [hunk 7]
```diff
@@ -74,13 +74,14 @@ const About = () => (
...
+                and educating more people.
```
Comment (marbar3778) [inline src/components/sections/About.js]:
```suggestion

```

File: src/components/sections/About.js [hunk 8]
```diff
@@ -74,13 +74,14 @@ const About = () => (
             <div>
               <h2>Build the Future</h2>
               <p>
-                We believe in helping to build out a future we would want ot live in. 
-                To help accomplish this goal we will be allotting 10% of our profits towards
-                OSS (open source software) this is not only limited to blockchain projects, beach and 
-                forest cleanups, and educating more people.
+                We believe in helping to build out a future we would want ot
+                live in. To help accomplish this goal we will be allotting 10%
+                of our profits towards OSS (open source software) this is not
```
Comment (marbar3778) [inline src/components/sections/About.js]:
```suggestion
                of our profits towards OSS (open source software), forest & beach cleanups and education.
```

File: src/components/sections/Faq.js [hunk 9]
```diff
@@ -10,30 +10,30 @@ const FAQS = [
...
     content: () => (
       <>
-        We provide safe and secure blockchain infrastructure for staking based blockchains.
-        We run validator services on a variety of different networks to secure blockchains
-        and earn rewards for running these services. When you stake with us we share the block rewards
-        with you.
+        We provide safe and secure blockchain infrastructure for staking based
+        blockchains. We run validator services on a variety of different
+        networks to secure blockchains and earn rewards for running these
+        services. When you stake with us we share the block rewards with you.
```
Comment (marbar3778) [inline src/components/sections/Faq.js]:
```suggestion
        services. 
```

File: src/components/sections/Faq.js [hunk 10]
```diff
@@ -10,30 +10,30 @@ const FAQS = [
...
     ),
   },
   {
     title: 'What is staking?',
     content: () => (
       <>
-        Staking is the process of holding funds in a cryptocurrency wallet to support opertions of a blockchain network.
-        It locks the curreny in a wallet to review rewards. The concept of staking is closely related to Proof of Stake.
-        You can learn more about that <ExternalLink href="https://en.wikipedia.org/wiki/Proof_of_stake">here.</ExternalLink>
+        Staking is the process of holding funds in a cryptocurrency wallet to
```
Comment (marbar3778) [inline src/components/sections/Faq.js]:
```suggestion
        Staking is part of the economic and security model of many networks.
```

File: src/components/sections/Faq.js [hunk 11]
```diff
@@ -10,30 +10,30 @@ const FAQS = [
...
+        support opertions of a blockchain network. It locks the curreny in a
```
Comment (marbar3778) [inline src/components/sections/Faq.js]:
```suggestion

```

File: src/components/sections/Faq.js [hunk 12]
```diff
@@ -10,30 +10,30 @@ const FAQS = [
...
+        wallet to review rewards. The concept of staking is closely related to
```
Comment (marbar3778) [inline src/components/sections/Faq.js]:
```suggestion

```

File: src/components/sections/Faq.js [hunk 13]
```diff
@@ -10,30 +10,30 @@ const FAQS = [
...
+        Proof of Stake. You can learn more about that{' '}
```
Comment (marbar3778) [inline src/components/sections/Faq.js]:
```suggestion
        If you'd like to learn more about staking and securing networks there is a nicely written article by Lunie.io, you can find it {' '}
```

File: gatsby-node.js (hunk 4 above)
Comment (foodaka) [reply gatsby-node.js]: yes we are using this to generate our blog posts see line 46

File: src/components/sections/Faq.js (hunk 12 above)
Comment (foodaka) [reply src/components/sections/Faq.js]: for all your comments around wording can you seperate them into a seperate pr - the only changes on those files are from me accidently running a format command on root

**Comment by foodaka** [comment]
@marbar3778 im going to merge it - before we release prod lets do 1 workthrough on the messaging. i have another PR open for that lets do that there

**Pull request merged**